components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.email/src/main/resources/org/wso2/carbon/event/output/adapter/email/i18n/Resources.properties

```diff
@@ -17,5 +17,10 @@
 #
 
 email.address=Email Address
+emailAddress.hint=Register publisher for multiple email IDs' using comma separated delimiter(",")
 email.subject=Subject
+email.type=Email Type
+emailType.hint= Select the email format to be sent
+
+
 
```

Comment:
It is being already removed in the commit named "Refactoring Email event adapter" (Commit ID bbba41b)